Library/Formula/nspr.rb

```diff
@@ -32,12 +32,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
```

Comment:
This chunk appears to be unnecessary. `--enable-macos-target=0` seems to have no ill effect on Linux.